test/scripts/openshift-ci/setup-e2e-tests.sh

```diff
@@ -90,10 +90,7 @@ fi
 
 if [[ "${DEPLOYMENT_PROFILE}" == "llm-d" ]]; then
   echo "⏳ Installing llm-d prerequisites"
-  $SCRIPT_DIR/infra/deploy.cert-manager.sh
-  $SCRIPT_DIR/infra/deploy.lws.sh
-  source $SCRIPT_DIR/infra/deploy.istio-exp.sh
-  install_upstream_istio "${PROJECT_ROOT}"
+  $SCRIPT_DIR/setup-llm.sh false
```

Comment:
By simply reading this line it is not clear what is the intent here.

Follow-up suggestion: make it a flag `--deploy-kserve` so it is self-documenting and not positional?